.vscode/launch.json

```diff
@@ -31,6 +31,40 @@
                "dataset_path=gs://test-maxtext-dataset",
                "steps=2",
                "enable_checkpointing=false"]
-    }
+    },
+    {
+      "name": "Debug MaxText Inference Microbenchmark",
+      "type": "python",
+      "request": "launch",
+      "console": "integratedTerminal",
+      "justMyCode": false,
+      "python": "python3",
+      "program": "${workspaceFolder}/MaxText/inference_microbenchmark.py",
+      "args": [
+        "MaxText/configs/base.yml",
+        "model_name=llama2-7b",
+        "tokenizer_path=assets/tokenizer.llama2",
+        "weight_dtype=bfloat16",
+        "scan_layers=false",
+        "attention=dot_product",
+        "max_prefill_predict_length=1024",
+        "max_target_length=2048",
+        "ici_fsdp_parallelism=1",
+        "ici_tensor_parallelism=-1",
+        "ici_autoregressive_parallelism=1",
+        "inference_microbenchmark_prefill_lengths=32,64,128,256,512,1024",
+        "inference_microbenchmark_stages=generate",
+        "inference_microbenchmark_loop_iters=1",
+        "run_name=runner_$(date +%Y-%m-%d-%H-%M)", 
+        "base_output_directory=gs://test-maxtext-output",
+        "prefill_cache_axis_order=0,2,1,3",
+        "ar_cache_axis_order=0,2,1,3",
+        "compute_axis_order=0,2,1,3",
+        "reshape_q=true",
+        "per_device_batch_size=24",
+        "quantization=int8",
+        "quantize_kvcache=True",
+      ]
+    },
```

Comment:
n00b question -- Is this to auto run for every local commit/amend on vscode? or just for convinience?

Reply:
no, I added this when I was debugging on my local vscode, which is helpful for other engineers, but you may need to change the flags depends on your run.